public/app/features/plugins/admin/components/Badges/PluginAngularBadge.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { PluginErrorCode } from '@grafana/data';
+import { Badge } from '@grafana/ui';
+
+type Props = { error?: PluginErrorCode };
+
+export function PluginAngularBadge({ error }: Props): React.ReactElement {
+  return <Badge icon="exclamation-triangle" text="Angular" color="orange" tooltip="Legacy Angular plugin" />;
```

Comment:
IMO the tooltip information here is lacking. not everyone understand what is angular or why is legacy.

What about making the tooltip: `Plugin's use of legacy Angular platform unsupported by Grafana may impact functionality.`

Reply:
Agree. @sympatheticmoose do you have any other suggestions about copy for this tooltip and https://github.com/grafana/grafana/pull/69084/files#r1227801227 ?

Reply:
For the tooltip, how about something fairly blunt:
> This plugin uses deprecated functionality, support for which is being removed.

Only concern is how well that sentence reads for folks where english isn't their first language. The tooltip we need to keep quite brief, but it is tricky to convey the nuance that it might already be incompatible, and definitely will be incompatible in the future.

Then for the warning text where we can be more verbose:
> This plugin uses a deprecated, legacy platform based on AngularJS and may be incompatible depending on your Grafana configuration.

Reply:
if there is an easy way to detect the value of the config setting for angular support and to dynamically change the text based on that, let me know and I'll rethink :)

Reply:
@sympatheticmoose I have updated the copy as suggested.
And yes, it would be fairly easy to do something like:

```js
This plugin uses a deprecated, legacy platform based on AngularJS and
        {angularSupportEnabled === undefined
          ? ' may be incompatible depending on your Grafana configuration.'
          : angularSupportEnabled
          ? ' will stop working in future releases of Grafana.'
          : ' is incompatible with your current Grafana configuration.'}
```

WDYT?

Reply:
Nice! I like your suggestion and agree if we want to change the edit panel text that can be a follow up

Reply:
Changed :smiley: